src/data/resolvers/motions.ts

```diff
@@ -158,6 +158,28 @@ export const motionsResolvers = ({
       const timeToSubmitMS = motion.events[1].toNumber() * 1000;
       const timeToSubmitInPast = timeToSubmitMS < blocktime;
 
+      const latestMotionStakedEvent = sortedEvents.find(
+        (event) => event.name === ColonyAndExtensionsEvents.MotionStaked,
+      );
+
+      if (latestMotionStakedEvent) {
+        // eslint-disable-next-line max-len
+        const totalStakeFraction = await votingReputationClient.getTotalStakeFraction();
+        const requiredStake = getMotionRequiredStake(
+          motion.skillRep,
+          totalStakeFraction,
+          18,
```

Comment:
These should be the token's actual decimals, why is this value hard coded ?

Reply:
We addressed this in #2420, I think. It's ok to leave it like this